components/elm/cime_config/config_component.xml

```diff
@@ -74,8 +74,7 @@
       <value compset="20TR.*_EAM%CMIP6_ELM">20thC_CMIP6_transient</value>
       <value compset="20TR(?:SOI)?_EAM%CMIP6_ELM*_BGC%B">20thC_CMIP6bgc_transient</value> 
       <value compset="20TR.*_ELM%[^_]*BGC">20thC_bgc_transient</value>
-      <value compset="1950(?:SOI)?_EAM%CMIP6-LR.*_ELM">1950_CMIP6LR_control</value>
-      <value compset="1950(?:SOI)?_EAM%CMIP6-HR_ELM">1950_CMIP6HR_control</value>
```

Comment:
Is the support for `CMIP6-LR_ELM` and `CMIP6-HR_ELM` being discontinued?

Reply:
Thanks for the careful review, @bishtgautam . Yes, they are discontinued. CMIP6-LR and CMIP6-HR only relevant for V1 code base.